src/components/line-input/styles/SLineInputLabel.ts

```diff
@@ -5,9 +5,14 @@ import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from "../../../consta
 const Bronze = css`
   ${CSSProgressiveCaption01Semibold};
   position: relative;
-  opacity: 0;
-  transition: opacity 200ms, color 200ms;
+  ${(props) =>
+    !props.isLabelVisible &&
```

Comment:
@yousseefben  can you share the use-case for this?

Reply:
because of the branch name there is no more link to linear so its hard for me to review if we need this

Reply:
ah i see, it's on the dopdownsaveview, the label is by default hidden, so on figma we should see the label here the [link](https://www.figma.com/file/XrUkpZwlWi2swxzvpte03J/Screens-%26-Flows?node-id=258%3A44200) to figma